README.md

```diff
@@ -101,6 +101,12 @@ Read more about the [Ory Elements E2E library](./packages/test/README.md).
 
 ## Breaking Changes
 
+### v0.0.1-beta.14
```

Comment:
maybe let's bump this version to a new major version
0.0.2-beta.1

Reply:
New major would be `1.0.0`, but :shrug: 
npm will also auto-update to ` 0.0.2-beta.1`

Reply:
ah true. but at least we indicate something changed between 0.0.1 -> 0.0.2

Reply:
or maybe we should bump to 0.1.0 ?